### DISCUSSION TITLE
The “suggested” interface defaults to returning questions in English.

### DISCUSSION THREAD

**Discussion by beardog6** [langgenius/dify#4774] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

I tried to use the suggested interface to return suggestions, but often receive suggestions in English. 

### 2. Describe the feature you'd like to see

The prompts for the system’s built-in interfaces are all written in English. It is hoped that the console can add an option for system language, and the prompts for the built-in interfaces can request that the large model return results in the corresponding language based on the system language setting.

### 3. How will this feature improve your workflow or experience?

Enhance the user experience for non-native English speakers.

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by patryk20120** [comment]
Do you use chatflow / workflow? Where do you get the results from?
